### ISSUE TITLE
[Bug]: During Combat, Combat Trackers becomes intrusive

### ISSUE THREAD

**Issue by rpganalyst** [arcanistzed/ctg#14]
### Expected Behavior

Combat Tracker is supposed to stay in the Combat Tracker tab.

### Current Behavior

Combat Tracker appears in the below half of the other tabs, during combat.

E.g. I'm reading the chat log and suddently it appears below, stealing space from the chat log tab.
The same with the other tabs.
If i go to the real Combat Tracker bar and then back to the Chat Log, the Combat Tracker's is gone away.

### Steps to Reproduce

1. Start a Combat with other users logged in.
2. Do something in a tab while they are using Items logging something to the Chat Log.
3. Look a the Combat Tracker tab appear everywhere in the tab you are in.

### Context

This does not happen if i run a combat while I'm testing alone.

### Version

0.1.0

### Foundry VTT Version

9.269

### Operating System

Windows 10

### Browser / App

Chrome

### Game System

dnd5e 1.6.3

### Modules Disabled

- [ ] I can confirm that disabling all of my other modules doesn't fix this issue.

**Comment by rpganalyst** [comment]
I can't confirm the **Modules Disabled** section because without the other players logged doing stuff I've no issues. And for me is not easy do tests during sessions.

**Comment by rpganalyst** [comment]
> Could you show me a screenshot?

Indeed, I've a game session planned for Tuesday.

**Comment by arcanistzed** [comment]
I'm fairly certain that this is the issue above, so I'll close this for now.

**Comment by rpganalyst** [comment]
> The reason I ask for a screenshot is that this sounds a lot like a core issue that I've experienced before with other modules. Does it look like this? ![image](https://user-images.githubusercontent.com/82790112/176926520-355b33e8-458b-46ef-bab1-338316079f9a.png)

Hello, sorry for the delay, but I was traveling. I was not able to reproduce it out of a real RPG session, so my screenshot will come on tuesday night.
Anyway the screenshot you attached is 100% identical to what I experienced.
How do you've handled It?

Thank you.

**Comment by arcanistzed** [comment]
It should go away if you switch sidebar tabs. There's not much I can do about it since it's a core issue (I reported it [here](https://github.com/foundryvtt/foundryvtt/issues/7420)), but it looks like they intend to fix it for v10 stable.

**Comment by rpganalyst** [comment]
> It should go away if you switch sidebar tabs. There's not much I can do about it since it's a core issue (I reported it [here](https://github.com/foundryvtt/foundryvtt/issues/7420)), but it looks like they intend to fix it for v10 stable.

Thank you for you time :)

**Comment by Bumsbirne** [comment]
Hi i know this is closed but i had the same bug and i could easily reproduce it. It always happens when the tracker comes to a singular combatant without a toggle beside it, so i rolled back to 1.0 where every combatant had a toggle and the problem went away.

**Comment by arcanistzed** [comment]
I tested the new fixes in v10 and they didn't fix this, so I was very confused. I spent three hours trying to figure it out before realizing that there's actually a second reason why this issue occurs and it's a very small bug in my code.
I'll be fixing that and then backporting the v10 patch to v9 users, so hopefully this shouldn't be an issue in the next release! 
Thank you all for the reports :smile:
